packages-node/providence-analytics/test-node/program/utils/swc-traverse.test.js

```diff
@@ -420,9 +429,17 @@ describe('swcTraverse', () => {
       expect(babelScopes.length).to.equal(swcScopes.length);
       for (let i = 0; i < babelScopes.length; i += 1) {
         expect(babelScopes[i].uid - babelRootScopeIdOffset).to.equal(swcScopes[i].id);
-        expect(Object.keys(babelScopes[i].bindings)).to.deep.equal(
-          Object.keys(swcScopes[i].bindings),
-        );
+
+        const babelEntries = Object.entries(babelScopes[i].bindings);
+        const swcEntries = Object.entries(swcScopes[i].bindings);
+        for (const [j, [bindingKey, binding]] of babelEntries.entries()) {
+          expect(bindingKey).to.equal(swcEntries[j][0]);
+          expect(binding.path.node.type).to.equal(swcEntries[j][1]?.path?.node.type);
+        }
+
+        // expect(Object.keys(babelScopes[i].bindings)).to.deep.equal(
```

Comment:
Can you add a todo to fix this expect?

Reply:
Ah, good one. Will remove this in a follow-up 👍